src/docs/product/cli/configuration.mdx

```diff
@@ -28,7 +28,7 @@ sentry-cli --url https://myserver.invalid/ login
 
 ## To authenticate manually:
 
-Visit your [auth token user settings page](https://sentry.io/settings/account/api/auth-tokens/) and create or copy an existing token. Then either:
+Visit the [Organization Auth Token Settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/auth-tokens/) and create a new token or copy an existing one. Then either:
```

Comment:
m: You can't copy an existing one anymore 😅 so:
```suggestion
Visit the [Organization Auth Token Settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/auth-tokens/) and create a new token. Then either:
```

Reply:
Ah right